lib/broccoli/ember-app.js

```diff
@@ -212,6 +229,11 @@ function EmberApp(options) {
   this._notifyAddonIncluded();
 }
 
+/**
+  Returns the environment name
+
+  @return {String} Environment name
```

Comment:
Should indicate that this is a class method (and does not require instantiation), via `@static` YUIDoc flag.